features/aborting.yml

```
@@ -2,3 +2,16 @@ name: AbortController and AbortSignal
 description: The `AbortController` and `AbortSignal` APIs allow you to cancel an ongoing operation, such as a `fetch()` request.
 spec: https://dom.spec.whatwg.org/#aborting-ongoing-activities
 caniuse: abortcontroller
+compat_features:
```

I looked at the additions to this and they make sense to me as later additions. However, the computed status now departs from Can I Use significantly (eventually we'll have docs for this, but see https://github.com/web-platform-dx/web-features/issues/1631#issuecomment-2310669265 for now). This needs a `compute_from` and a dist refresh:

```suggestion
status:
  compute_from: api.AbortController
compat_features:
```

---

hmmm yeah i need to add that to my checklist